remo/__init__.py

```diff
@@ -1,163 +1,41 @@
+def connect():
+    """
+    Connect to a local running remo server.
     
-    Failed to launch Remo server, please start it manually.
-    
-    To fix config issue, please run init or update for Remo server (inside Remo conda env), like: 
-    python -m remo_app init
-    or
-    python -m remo_app update
-    """.format(
-                        config.conda_env
-                    )
-                )
-                return False
-
-            global _logs
-            global _server
-            _logs = open(str(os.path.join(REMO_HOME, 'logs.txt')), 'w')
-
-            path_args = ['python'] if os.name == 'nt' else ['bin', 'python']
-            python_exe = os.path.join(config.conda_env, *path_args)
+    """
+    from .config import Config
+    config = Config.load()
+    connect_sdk(config.server_url(), config.user_email, config.user_password, config.viewer)
 
-            import subprocess
 
-            _server = subprocess.Popen(
-                '{} -m remo_app'.format(python_exe),
-                stdout=_logs,
-                stderr=_logs,
-                shell=True,
-                universal_newlines=True,
-            )
-
-            time.sleep(5)
-            retry = 5
-            while retry > 0:
-                print('Wait a bit... ', retry)
-                try:
-                    resp = requests.get(version_endpoint)
-                    print(
-                        """
-    (\(\ 
-    (>':') Remo server is running: {}
-                    """.format(
-                            resp.json()
-                        )
-                    )
-                    if open_browser:
-                        BrowserViewer().browse(server_url)
-
-                    return True
-                except:
-                    time.sleep(1)
-                retry -= 1
-
-            print('Failed to launch Remo server, please start it manually')
-            return False
-
-        launch_server(open_browser=False)
-        global _sdk
-        from .sdk import SDK
-
-        _sdk = SDK(server_url, config.user_email, config.user_password, viewer)
-
-        # set access to public SDK methods
-        import sys
-
-        is_public_sdk_method = lambda name: not name.startswith('_') and callable(getattr(_sdk, name))
-        functions = filter(is_public_sdk_method, dir(_sdk))
-        for name in functions:
-            setattr(sys.modules[__name__], name, getattr(_sdk, name))
-
-        setattr(sys.modules[__name__], 'launch_server', launch_server)
-        setattr(sys.modules[__name__], 'terminate_server', terminate_server)
-
-    else:
-        print(
-            """
-        WARNING: Config not found in {}
-        
-        Please run init for Remo server (inside Remo conda env), like: 
-        
-        python -m remo_app init
-        
-        """.format(
-                cfg_path
-            )
-        )
-
-
-__init__()
+def connect_sdk(server: str, email: str, password: str, viewer: str = 'browser'):
```

Comment:
@vmanilo do we want to use the default viewer from the config file?

Or are you thinking that if installing the sdk alone, user won't have the config file?

Reply:
@drewlr I think it's good how you did

if user don't have config file, `init()` will fail, but user can use `connect_sdk()`